authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc

```diff
@@ -64,6 +64,10 @@ Mint mode is the default and recommended best practice setting for the CCO to us
 |
 
 |====
+[.small]
+--
+1. Manual mode is the only supported CCO configuration for Microsoft Azure Stack Hub.
```

Comment:
If manual is the only supported mode, then why are there X's in the columns for the other two options for Azure? Or is "Stack Hub" something specific and there are other Azure options that these are supported for?